src/main/java/io/github/sefiraat/crystamaehistoria/listeners/MiscListener.java

```diff
@@ -43,6 +47,16 @@ public void onBlockPlacerStoriedBlock(BlockPlacerPlaceEvent e) {
         }
     }
 
+    @EventHandler
+    public void onShootPaintbrush(EntityShootBowEvent e) {
+        ItemStack itemStack = e.getConsumable();
+        if (SlimefunItem.getByItem(itemStack) instanceof BasicPaintbrush || SlimefunItem.getByItem(itemStack) instanceof InfinitePaintbrush) {
```

Comment:
Both classes implement `MagicPaintBrush` so that can be used here instead of two checks. Also, for the future, if you do need to do two checks like this, you should declare the SlimefunItem above so you don’t call getByItem() twice. But changing to magic paintbrush will resolve this :)

Reply:
Alright there we go